### PR TITLE
Move logging flags to separate configuration class

### PR DESCRIPTION
**Pull request by neumayer**

### CODE REVIEW

**Comment by acidmoose** [comment]
Isn't this the same change as #36 ? I'm confused.

File: jsonlogging/src/main/java/org/secondbase/logging/JsonLoggerConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package org.secondbase.logging;
+
+import org.secondbase.flags.Flag;
+
+public class JsonLoggerConfiguration {
+
+    @Flag(name = "environment", description = "The environment the service runs in (test|staging|production).", required = false)
```
Comment (acidmoose) [inline jsonlogging/src/main/java/org/secondbase/logging/JsonLoggerConfiguration.java]: long line

File: jsonlogging/src/main/java/org/secondbase/logging/JsonLoggerConfiguration.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    public static String environment;
+
+    @Flag(name = "service", description = "The name of the service (testservice).", required = false)
```
Comment (acidmoose) [inline jsonlogging/src/main/java/org/secondbase/logging/JsonLoggerConfiguration.java]: required=false is default and can be omitted.

**Comment by neumayer** [comment]
I'm also a bit confused now.

**Pull request closed without merging**